teb_local_planner/src/optimal_planner.cpp

```diff
@@ -106,6 +106,7 @@ void TebOptimalPlanner::initialize(nav2_util::LifecycleNode::SharedPtr node, con
   vel_goal_.second.linear.y = 0;
   vel_goal_.second.angular.z = 0;
   initialized_ = true;
+  setVisualization(visual);
```

Comment:
is this related?

Reply:
It was in the commit cherry-picked from upstream